[UI] Full Dark Mode

Vel-San opened the pull request:
Hey ^_^

I'm using your _Moving_ theme and loved it, so I decided to make a Dark version of it to reduce eye restraint! 

I also have several other fixes and improvements/Add-ons that I'm using and will create PRs for them separately.

If interested, you can see it in action here: ~https://vel-san.github.io/~ https://bleeding-action-man.github.io/

Thanks for your work!

Vel-San commented:
If interested, you can see it in action here: https://vel-san.github.io/

huangyz0918 commented:
Wow, cool, I love this theme @Vel-San ! Can you update a theme trigger in the `_config.yml` file? Current changes will overwrite the light theme, so I cannot merge it.

You can add an attribute like `dark-mode: true` in the under the moving label, and load the different css according to that, if you have any issues in this feature, I'm glad to help!

Vel-San commented:
@huangyz0918 Thanks!

Yes of course, personally i wanted to make a button that alternates between dark/light mode. I didn't know about the trigger, i'll do it :) !

huangyz0918 commented:
> I didn't know about the trigger, i'll do it :) !

Since the Jekyll is a static blog framework, a dynamic button is inappropriate here. Actually it is a really simple feature, you can refer to the [Liquid Language](https://shopify.github.io/liquid/basics/introduction/), that builds Jekyll theme.

Vel-San commented:
~~@huangyz0918 Do you know if there's a way to override colors in scss based on the trigger BOOL in the _config ? Or can i override colors in the html ? will the classes take the new overridden variable? if not, i think i'll be duplicating each class into light and dark .. I feel this isn't the proper solution.~~

I'm so stupid! I don't know why i tried to make it so complex for nothing 😄 !

Issue should be fixed now. Default is Light theme!

Vel-San commented:
Note: I still need to fix the difference between Dates and Post titles ( Grey and White / Grey and Black )

I'll release a fix soon :)

Vel-San commented:
Done! I've fixed all mismatching colors. Previosly everything was set on text-color, i've changed it for better readability and eye-friendlly between Black/Grey/White.

Fixes applied on both Light and Dark modes. I also published them on my own version here as well  

https://vel-san.github.io/

huangyz0918 commented on README.md:
```diff
@@ -36,6 +36,12 @@ jekyll serve
 
+**Note:** If you want to switch between dark and light themes, navigate to
+
+> assets \> main.scss
```
You know, the only place you can put the trigger is `_config.yml`, since many people don't have `assets \> main.scss`. You can refer to https://github.com/huangyz0918/personal-page-blog, how can I change the theme? I don't know where is the `main.scss`.

Vel-San replied:
I made the change. Would that do the trick? The trigger is now in confg file, you just need to switch between _sass or _sass-dark in _config.yml
